migtests/scripts/functions.sh

```diff
@@ -456,7 +456,6 @@ import_data_file() {
 		--target-db-host ${TARGET_DB_HOST} \
 		--target-db-port ${TARGET_DB_PORT} \
 		--target-db-user ${TARGET_DB_USER} \
-		--target-db-schema "public" \
```

Comment:
is the default "public" if not specified? 
Should we maybe use an env var SIMILAR to the other args?

Reply:
> is the default "public" if not specified?
yes its default public